sdb/internal/cli.py

```diff
@@ -120,6 +120,15 @@ def parse_arguments() -> argparse.Namespace:
     #
     if args.object and not args.core:
         parser.error("raw object file target is not supported yet")
+
+    #
+    # For the case where the -s option is not specified (e.g.
+    # sdb <binary> <dump>) we instantiate symbol_search ourselves
+    # so it has a type even though it is an empty list.
+    #
+    if not args.symbol_search:
+        args.symbol_search = []
```

Comment:
When adding the "symbol-search" option above, it looks like we define it to be a "str", so it strikes me as odd that we'd initialize it to an empty list here, rather than an empty string..

Can you comment on why the types differ? Can we consistently use the same type for both cases?

Also, could we more simply set a "default" value when one is not specified on the command like, using argparse directly? i.e. use `default=...` when we create/add that argument via `dis_group.add_argument`.

Reply:
You are right, I'll give it another look soon.